components/src/dynamo/vllm/tests/omni/test_omni_realtime_handler.py

```
@@ -0,0 +1,316 @@
+import asyncio
+import base64
+from types import SimpleNamespace
+
+import numpy as np
+import pytest
+
+try:
+    # Importing the omni package pulls omni_handler -> vllm_omni; the handler
+    # logic itself is vllm-free, but the package import is not.
+    from dynamo.vllm.omni.realtime_handler import RealtimeOmniHandler
+except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)